openassessment/xblock/test/test_submission.py

```diff
@@ -212,11 +212,10 @@ def test_remove_all_uploaded_files(self, xblock):
             course_id='test_course',
             anonymous_student_id='test_student',
         )
-
         download_url = api.get_download_url("test_student/test_course/" + xblock.scope_ids.usage_id)
         resp = self.request(xblock, 'download_url', json.dumps(dict()), response_format='json')
         self.assertTrue(resp['success'])
-        self.assertEqual(download_url, resp['url'])
+        self.assertTrue(resp['url'].startswith(download_url))
```

Comment:
I updated the `assertEqual`  to `assertTrue ` because the test becomes flaky due to the query string in the `download_url`. Query string parameters are time-dependent (expiration and signature)